packages/server/src/ChildProcess.ts

```diff
@@ -83,6 +83,7 @@ export class ChildProcess {
                 depthQueue,
                 componentNodes,
                 incomingInput.question,
+                '',
```

Comment:
can we also pass in the `chatid` from here - https://github.com/FlowiseAI/Flowise/blob/main/packages/server/src/index.ts#L445 ?

Reply:
It's missing, I have forgot the `chatflow` variable contains a correct chatflow.id we can use, I'll fix it later!👀